bytebase/bytebase.com
# feat: docs sidebar, update logos

This pull request brings some changes:

1. Sticky docs navigation
2. Styling logos
3. Refactor mobile sidebar

**Steps to check:**

- open previews of docs, home page, contact, 
- make sure that everything looks and works as expected
